odata-protocol/11.4 Data Modification.md

```diff
@@ -602,8 +602,10 @@ annotating the singleton with the term `Capabilities.DeleteRestrictions`
 (nested property `Deletable` with value `true`) defined in
 [OData-VocCap](#ODataVocCap).
 
-On successful completion of the delete, the response MUST be
-[`204 No Content`](#ResponseCode204NoContent) and contain an empty body.
+On successful completion of the delete, the response MUST either be
+[`204 No Content`](#ResponseCode204NoContent) and contain an empty body,
+or [`200 OK`](#ResponseCode200OK) and contain a representation of a
+`deleted entity` according to the specified format.
```

Comment:
```suggestion
deleted entity according to the specified format.
```
Or why was that in monospace?

Reply:
I checked in Protocol and JSON Format and we consistently use standard font for "deleted entity".